media/js/src/firstOrderLogic/gridTemplates/hardTemplates/hardTemplate_06.ts

```diff
@@ -49,11 +49,11 @@ export const hardTemplate_06 = {
         );
 
         // FOL example:
-        // "∀x( (Shape(x)=s1 ∧ Color(x)=c1) → ∃y( Shape(y)=s2 ∧ Value(y)<threshold ∧ TopRightDiagonalOf(y,x) ) )"
+        // "∀x( (Shape(x)=s1 ∧ Color(x)=c1) → ∃y( Shape(y)=s2 ∧ Value(y)<threshold ∧ TopRightOf(y,x) ) )"
         const formalFOLStatement = `
         ∀x (
           (Shape(x, ${shape1}) ∧ Color(x, ${colorName}))
-          → ∃y (Shape(y, ${shape2}) ∧ Value(y) < ${threshold} ∧ TopRightDiagonalOf(y, x))
+          → ∃y (Shape(y, ${shape2}) ∧ Value(y) < ${threshold} ∧ TopRightOf(y, x))
```

Comment:
I am curious about the change here. Can you explain?

Reply:
Honestly, I just felt like it was wordy, and that can present a higher risk of typos that frustrate the user. Identifying it as a diagonal didn't feel necessary to describe the relationship. I can revert it if you would prefer.